traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/models/Pass.java

```diff
@@ -47,40 +44,27 @@ public Price getPrice() {
         return price;
     }
 
+    public PurchasePass toPurchasePass() {
```

Comment:
See my comment on adding a comment indicating future versions of ourselves (or other developers) what's going on here.